Source/WebKit/GPUProcess/media/RemoteMediaPlayerManagerProxy.messages.in

```diff
@@ -23,6 +23,7 @@
 
 #if ENABLE(GPU_PROCESS) && ENABLE(VIDEO)
 
+[EnabledBy=MediaPlaybackEnabled]
```

Comment:
UseGPUProcessForMediaEnabled for all the media related GPU process proxies is probably more accurate as well.
Or maybe UseGPUProcessForMediaEnabled & MediaPlaybackEnabled if we have a case where would set the first but not the second (I haven't checked that).